man/docker-create.1.md

```diff
@@ -240,6 +242,22 @@ This value should always larger than **-m**, so you should always use this with
 **-w**, **--workdir**=""
    Working directory inside the container
 
+# EXAMPLES
+
+## Running a container with --interactive and without attaching to STDIN
```

Comment:
Is there a use case for this because I'm not sure this is something that we would even want to add to the docs.  

It's like saying I would like to interact with this computer but don't plugin the keyboard.